contents/uix/design-systems.md

```diff
@@ -0,0 +1,130 @@
+
+The typical structure of a Design System looks like this:
+<center>
+<img src="https://cdn-images-1.medium.com/max/2600/1*wSHUJh069L618oXqVZYDtA.png" alt="MySQL Workbench GUI Software" width="95%">
+
+_Figure 1. Typical structure of a Design System_ ([source](https://uxdesign.cc/can-design-systems-fix-the-relationship-between-designers-developers-eb12fc9329ab))
+</center>
+
+Below is a brief explanation of the parts that make up a Design System. The explanation includes reference to a living example, [Polaris](https://polaris.shopify.com/), Shopify's Design System.
+- `Rules`: Rules include abstract elements such as brand values, mindset, and shared beliefs. You can refer to Polaris' [product experience principles](https://polaris.shopify.com/patterns-and-guides/product-experience-principles#navigation).
+- `Pattern Library`: A Pattern Library integrates functional components and their usage. You can refer to Shopify's [product component library](https://github.com/Shopify/polaris-react).
+- `Building Blocks`: Building Blocks refer to documentation consisting of the collection of interface elements. You can refer to Polaris' documentation which describes their [components](https://polaris.shopify.com/components/get-started).
+- `Style Guide`: A Style Guide focuses on graphic styles such as colors, fonts, illustrations, etc and their usage. You can refer to Polaris' [style guide](https://polaris.shopify.com/design/colors).
+
+## Why should we use a Design System?
+
+<center>
+<img src="https://cdn-images-1.medium.com/max/2600/1*wFOwLNi-xGBfyGnssynQaQ.png" alt="Typical Design and Development process" width="95%">
+
+_Figure 2. Typical Design and Development process_ ([source](https://uxdesign.cc/can-design-systems-fix-the-relationship-between-designers-developers-eb12fc9329ab))
+</center>
+
+Consider the typical design and development process. Designers come up with sketches, wireframes and hand them over to the developers to implement them through code. Here's the problem.
+
+Designers are looking at multiple directions and solutions, constantly comparing and tweaking their design. On the other hand, developers are ultimately accountable for shipping code. They focus on making sure that the correct architecture is in place, thinking about security and the performance of the product, hoping that the designers already have all the design details thought out.
+
+Over time, it is likely that many teams working on different parts of the product will create inconsistencies in the product. For example, HubSpot [explained how they found inconsistencies](https://product.hubspot.com/blog/how-building-a-design-system-empowers-your-team-to-focus-on-people-not-pixels) in their interface after auditing their User Interface (UI) components.
```

Comment:
Be more specific about what kind of _inconsistencies_ we are referring to here? UI/UX design inconsistencies?